interface/modules/zend_modules/module/Patientvalidation/view/layout/admin.phtml

```diff
@@ -0,0 +1,4 @@
+<main>
```

Comment:
Need a standard header. See here for an example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock